### ISSUE TITLE
Xcode 13.3 beta prints warning `Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS`

### ISSUE THREAD

**Issue by maxelerate** [flutter/flutter#98298]
## Steps to Reproduce

Issues with Macbook Air M1 chip & Flutter Firebase
<img width="198" alt="Screen Shot 2022-02-11 at 3 18 28 PM" src="https://user-images.githubusercontent.com/99514834/153683569-ebd680e3-b136-4524-a968-ba4ec6d4e591.png">

Flutter 2.10.1 • channel stable • https://github.com/flutter/flutter.git
Framework • revision db747aa133 (2 days ago) • 2022-02-09 13:57:35 -0600
Engine • revision ab46186b24
Tools • Dart 2.16.1 • DevTools 2.9.2

```
Flutter Doctor -verbose
flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.10.1, on macOS 12.0.1 21A559 darwin-arm, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
[✓] Xcode - develop for iOS and macOS (Xcode 13.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.1)
[✓] Android Studio (version 2021.1)
[✓] VS Code (version 1.64.0)
[✓] VS Code (version 1.64.0)
[✓] Connected device (2 available)
[✓] HTTP Host Availability

• No issues found!

flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.10.1, on macOS 12.0.1 21A559 darwin-arm, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
[✓] Xcode - develop for iOS and macOS (Xcode 13.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.1)
[✓] Android Studio (version 2021.1)
[✓] VS Code (version 1.64.0)
[✓] VS Code (version 1.64.0)
[✓] Connected device (2 available)
[✓] HTTP Host Availability

• No issues found!
```

1. Android Studio version
<img width="488" alt="Screen Shot 2022-02-11 at 3 20 55 PM" src="https://user-images.githubusercontent.com/99514834/153683715-9c613d71-406a-40fa-88a0-447144351eb0.png">

2. Example : https://firebase.google.com/codelabs/firebase-get-to-know-flutter#5

```
Xcode build done.                                            2.0s
Failed to build iOS app
Error output from Xcode build:
↳
    2022-02-11 15:07:33.077 xcodebuild[63781:306900] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
    2022-02-11 15:07:33.077 xcodebuild[63781:306900] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
    xcodebuild: error: Could not resolve package dependencies:
      failed extracting 'https://dl.google.com/firebase/ios/swiftpm/8.12.0/GoogleAppMeasurement.zip' which is required by binary target 'GoogleAppMeasurement':
```

Summary: After many attempts with brew install cocoapods and installing Xcode Beta + Flutter, it appears there are issues with Flutter SDK package
https://dl.google.com/firebase/ios/swiftpm/8.12.0/GoogleAppMeasurement.zip'

Please help with steps on how to cross this error with GoogleAppMeasurement.zip in Xcode.

Thanks,
Maxelerate

**Comment by darshankawar** [comment]
@maxelerate 
Do you get the errors as you reported only after following the steps mentioned in the firebase codelabs ? or in other projects as well ?
Can you create a new project and run it to see if you get similar errors with it too or not ?

I haven't looked at the codelab part yet, but does it mention to add any specific dependencies ?

**Comment by jinfagang** [comment]
@darshankawar @maxelerate  I got same error after I just upgraded to Xcode 13.3-beta.

```
Error output from Xcode build:
↳
    2022-03-02 17:45:38.148 xcodebuild[62848:6695836] Requested but did not find extension point with identifier
    Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in
    com.apple.dt.IDEWatchSupportCore
    2022-03-02 17:45:38.148 xcodebuild[62848:6695836] Requested but did not find extension point with identifier
    Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of
    plug-in com.apple.dt.IDEWatchSupportCore
    ** BUILD FAILED **
```

It was almost same error. I think it's not about firebase (sorry for reply here), but I don't know how to resolve this error.

**Comment by afoone** [comment]
Try to run with rosetta

**Comment by js1972** [comment]
Anyone have an answer to this? Not sure why its been closed. I get this error on an Intel mac.

**Comment by darshankawar** [comment]
@js1972 
Since the original reporter has not answered, would you be able to provide more details to replicate this error by providing your `flutter doctor -v` ?

**Comment by js1972** [comment]
Hi @darshankawar. Sure no problems. Here's my output of `flutter doctor -v`:

```
❯ flutter doctor -v
[✓] Flutter (Channel stable, 2.10.3, on macOS 12.2.1 21D62 darwin-x64, locale en-AU)
    • Flutter version 2.10.3 at /Users/I540401/dev/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 7e9793dee1 (2 weeks ago), 2022-03-02 11:23:12 -0600
    • Engine revision bd539267b4
    • Dart version 2.16.1
    • DevTools version 2.9.2

[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.


[✓] Xcode - develop for iOS and macOS (Xcode 13.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).

[✓] VS Code (version 1.65.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.36.0

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 99.0.4844.74

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 2 categories.
```

Note that I do not need to build for Android which is why I have not installed Android Studio.

**Comment by darshankawar** [comment]
@js1972 
Thanks, but can you also provide steps to replicate if any, as when does it throw the said error ?

**Comment by js1972** [comment]
The only thing that has changed since last successful run is that I recently updated my mac so now on macos 12.2.1.
I tried `flutter run` and it failed.

So I then did the following:

- flutter clean
- rm -rf ios/Pods
- rm -rf ios/.symlinks
- rm  ios/Podfile
- rm ios/Podfile.lock
- pod cache clean --all
- flutter run

All the above mulitple times etc. All with no luck.

Another person on the other issue #100161 mentioned to set the ios deployment target to 15.3 (a recent version) but that also made no difference to me.

**Comment by tahaafg20** [comment]
Okay, just found a solution which might sound super dumb but updating my Mac OS to Moterey 12.3 solved the issue with no change in the code required.

**Comment by darshankawar** [comment]
I recently upgraded my macOS from Catalina to Monterey 12.2.1 and have not seen such issue while trying to run my apps. My Xcode version is 13.2.1, so I am not sure what would exactly be causing the issue.

I am going ahead and labeling it for further investigation from the team.

My macOS `12.2.1 21D62`

Also, this doesn't seem to be specific to M1, as I've seen same reports happening on Intel chip too.

**Comment by js1972** [comment]
I just deleted my project and re-cloned from github to try again. Got a little bit more error info:

` /Users/I540401/dev/smartguardian_app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.4.99. (in target 'Reachability' from project 'Pods')
    Result bundle written to path:`

```
Launching lib/main.dart on iPhone 12 Pro in debug mode...
Xcode build done.                                           51.7s
Failed to build iOS app
Error output from Xcode build:
↳
    2022-03-18 09:05:16.589 xcodebuild[48266:12403744] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
    2022-03-18 09:05:16.590 xcodebuild[48266:12403744] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
    ** BUILD FAILED **
Xcode's output:
↳
    Writing result bundle at path:
    	/var/folders/82/y_h0djv17yvdwh0w89577w080000gn/T/flutter_tools.U4bujC/flutter_ios_build_temp_dirzJ47kt/temporary_xcresult_bundle
/Users/I540401/dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_app_badger-1.3.0/ios/Classes/FlutterAppBadgerPlugin.m:17:9: warning: 'UIUserNotificationSettings' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNNotificationSettings [-Wdeprecated-declarations]
            UIUserNotificationSettings* notificationSettings = [UIUserNotificationSettings settingsForTypes:UIUserNotificationTypeAlert | UIUserNotificationTypeBadge | UIUserNotificationTypeSound categories:nil];
            ^
    In module 'UIKit' imported from /Users/I540401/dev/smartguardian_app/ios/Pods/Target Support Files/flutter_app_badger/flutter_app_badger-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:44:12: note: 'UIUserNotificationSettings' has been explicitly marked deprecated here
    @interface UIUserNotificationSettings : NSObject
               ^
/Users/I540401/dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_app_badger-1.3.0/ios/Classes/FlutterAppBadgerPlugin.m:17:105: warning: 'UIUserNotificationTypeAlert' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNAuthorizationOptions [-Wdeprecated-declarations]
            UIUserNotificationSettings* notificationSettings = [UIUserNotificationSettings settingsForTypes:UIUserNotificationTypeAlert | UIUserNotificationTypeBadge | UIUserNotificationTypeSound categories:nil];
                                                                                          ^
    In module 'UIKit' imported from /Users/I540401/dev/smartguardian_app/ios/Pods/Target Support Files/flutter_app_badger/flutter_app_badger-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:17:32: note: 'UIUserNotificationType' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, UIUserNotificationType) {
                                   ^
/Users/I540401/dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_app_badger-1.3.0/ios/Classes/FlutterAppBadgerPlugin.m:17:135: warning: 'UIUserNotificationTypeBadge' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNAuthorizationOptions [-Wdeprecated-declarations]
            UIUserNotificationSettings* notificationSettings = [UIUserNotificationSettings settingsForTypes:UIUserNotificationTypeAlert | UIUserNotificationTypeBadge | UIUserNotificationTypeSound categories:nil];
                                                                                          ^
    In module 'UIKit' imported from /Users/I540401/dev/smartguardian_app/ios/Pods/Target Support Files/flutter_app_badger/flutter_app_badger-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:17:32: note: 'UIUserNotificationType' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, UIUserNotificationType) {
                                   ^
/Users/I540401/dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_app_badger-1.3.0/ios/Classes/FlutterAppBadgerPlugin.m:17:165: warning: 'UIUserNotificationTypeSound' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNAuthorizationOptions [-Wdeprecated-declarations]
            UIUserNotificationSettings* notificationSettings = [UIUserNotificationSettings settingsForTypes:UIUserNotificationTypeAlert | UIUserNotificationTypeBadge | UIUserNotificationTypeSound categories:nil];
                                                                                          ^
    In module 'UIKit' imported from /Users/I540401/dev/smartguardian_app/ios/Pods/Target Support Files/flutter_app_badger/flutter_app_badger-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:17:32: note: 'UIUserNotificationType' has been explicitly marked deprecated here
    typedef NS_OPTIONS(NSUInteger, UIUserNotificationType) {
                                   ^
/Users/I540401/dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_app_badger-1.3.0/ios/Classes/FlutterAppBadgerPlugin.m:17:88: warning: 'UIUserNotificationSettings' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's UNNotificationSettings [-Wdeprecated-declarations]
            UIUserNotificationSettings* notificationSettings = [UIUserNotificationSettings settingsForTypes:UIUserNotificationTypeAlert | UIUserNotificationTypeBadge | UIUserNotificationTypeSound categories:nil];
                                                                                          ^
    In module 'UIKit' imported from /Users/I540401/dev/smartguardian_app/ios/Pods/Target Support Files/flutter_app_badger/flutter_app_badger-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIUserNotificationSettings.h:44:12: note: 'UIUserNotificationSettings' has been explicitly marked deprecated here
    @interface UIUserNotificationSettings : NSObject
               ^
/Users/I540401/dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_app_badger-1.3.0/ios/Classes/FlutterAppBadgerPlugin.m:19:44: warning: 'registerUserNotificationSettings:' is deprecated: first deprecated in iOS 10.0 - Use UserNotifications Framework's -[UNUserNotificationCenter requestAuthorizationWithOptions:completionHandler:] and -[UNUserNotificationCenter setNotificationCategories:] [-Wdeprecated-declarations]
            [[UIApplication sharedApplication] registerUserNotificationSettings:notificationSettings];
                                               ^
    In module 'UIKit' imported from /Users/I540401/dev/smartguardian_app/ios/Pods/Target Support Files/flutter_app_badger/flutter_app_badger-prefix.pch:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.4.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIApplication.h:251:1: note: 'registerUserNotificationSettings:' has been explicitly marked deprecated here
    - (void)registerUserNotificationSettings:(UIUserNotificationSettings *)notificationSettings API_DEPRECATED("Use UserNotifications Framework's -[UNUserNotificationCenter requestAuthorizationWithOptions:completionHandler:] and -[UNUserNotificationCenter setNotificationCategories:]", ios(8.0, 10.0)) API_UNAVAILABLE(tvos);
    ^
    6 warnings generated.
    error: the following command failed with exit code 0 but produced no further output
CompileC /Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/flutter_app_badger.build/Objects-normal/arm64/FlutterAppBadgerPlugin.o /Users/I540401/dev/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_app_badger-1.3.0/ios/Classes/FlutterAppBadgerPlugin.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'flutter_app_badger' from project 'Pods')
    /Users/I540401/dev/smartguardian_app/ios/Pods/Reachability/Reachability.m:465:63: warning: cast to smaller integer type 'unsigned int' from 'Reachability *' [-Wpointer-to-int-cast]
                                 NSStringFromClass([self class]), (unsigned int) self, [self currentReachabilityFlags]];
                                                                  ^~~~~~~~~~~~~~~~~~~
    1 warning generated.
    /Users/I540401/dev/smartguardian_app/ios/Pods/Reachability/Reachability.m:465:63: warning: cast to smaller integer type 'unsigned int' from 'Reachability *' [-Wpointer-to-int-cast]
                                 NSStringFromClass([self class]), (unsigned int) self, [self currentReachabilityFlags]];
                                                                  ^~~~~~~~~~~~~~~~~~~
    1 warning generated.
    While building module 'FirebaseMessaging' imported from /Users/I540401/dev/smartguardian_app/ios/Pods/Headers/Public/Firebase/Firebase.h:66:
    In file included from <module-includes>:1:
    In file included from /Users/I540401/dev/smartguardian_app/ios/Pods/Target Support Files/FirebaseMessaging/FirebaseMessaging-umbrella.h:13:
    In file included from /Users/I540401/dev/smartguardian_app/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FirebaseMessaging/FirebaseMessaging.h:18:
    /Users/I540401/dev/smartguardian_app/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessagingExtensionHelper.h:37:38: warning: 'UNMutableNotificationContent' is only available on iOS 10.0 or newer [-Wunguarded-availability]
    - (void)populateNotificationContent:(UNMutableNotificationContent *)content
                                         ^
    In module 'UserNotifications' imported from /Users/I540401/dev/smartguardian_app/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessagingExtensionHelper.h:22:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.4.sdk/System/Library/Frameworks/UserNotifications.framework/Headers/UNNotificationContent.h:97:12: note: 'UNMutableNotificationContent' has been marked as being introduced in iOS 10.0 here, but the deployment target is iOS 9.0.0
    @interface UNMutableNotificationContent : UNNotificationContent
               ^
    While building module 'FirebaseMessaging' imported from /Users/I540401/dev/smartguardian_app/ios/Pods/Headers/Public/Firebase/Firebase.h:66:
    In file included from <module-includes>:1:
    In file included from /Users/I540401/dev/smartguardian_app/ios/Pods/Target Support Files/FirebaseMessaging/FirebaseMessaging-umbrella.h:13:
    In file included from /Users/I540401/dev/smartguardian_app/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FirebaseMessaging/FirebaseMessaging.h:18:
    /Users/I540401/dev/smartguardian_app/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessagingExtensionHelper.h:37:1: note: annotate 'populateNotificationContent:withContentHandler:' with an availability attribute to silence this warning
    - (void)populateNotificationContent:(UNMutableNotificationContent *)content
    ^
    /Users/I540401/dev/smartguardian_app/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessagingExtensionHelper.h:38:47: warning: 'UNNotificationContent' is only available on iOS 10.0 or newer [-Wunguarded-availability]
                     withContentHandler:(void (^)(UNNotificationContent *_Nonnull))contentHandler;
                                                  ^
    In module 'UserNotifications' imported from /Users/I540401/dev/smartguardian_app/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessagingExtensionHelper.h:22:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.4.sdk/System/Library/Frameworks/UserNotifications.framework/Headers/UNNotificationContent.h:37:12: note: 'UNNotificationContent' has been marked as being introduced in iOS 10.0 here, but the deployment target is iOS 9.0.0
    @interface UNNotificationContent : NSObject <NSCopying, NSMutableCopying, NSSecureCoding>
               ^
    While building module 'FirebaseMessaging' imported from /Users/I540401/dev/smartguardian_app/ios/Pods/Headers/Public/Firebase/Firebase.h:66:
    In file included from <module-includes>:1:
    In file included from /Users/I540401/dev/smartguardian_app/ios/Pods/Target Support Files/FirebaseMessaging/FirebaseMessaging-umbrella.h:13:
    In file included from /Users/I540401/dev/smartguardian_app/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FirebaseMessaging/FirebaseMessaging.h:18:
    /Users/I540401/dev/smartguardian_app/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessagingExtensionHelper.h:37:1: note: annotate 'populateNotificationContent:withContentHandler:' with an availability attribute to silence this warning
    - (void)populateNotificationContent:(UNMutableNotificationContent *)content
    ^
    2 warnings generated.
    2 warnings generated.
    error: the following command failed with exit code 0 but produced no further output
CompileC /Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/firebase_analytics.build/Objects-normal/x86_64/FLTFirebaseAnalyticsPlugin.o /Users/I540401/dev/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_analytics-8.3.4/ios/Classes/FLTFirebaseAnalyticsPlugin.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'firebase_analytics' from project 'Pods')
    In file included from /Users/I540401/dev/smartguardian_app/ios/ImageNotification/NotificationService.m:10:
    /Users/I540401/dev/smartguardian_app/build/ios/Debug-iphonesimulator/FirebaseMessaging/FirebaseMessaging.framework/Headers/FirebaseMessaging.h:17:9: warning: double-quoted include "FIRMessaging.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #import "FIRMessaging.h"
            ^~~~~~~~~~~~~~~~
            <FirebaseMessaging/FIRMessaging.h>
    /Users/I540401/dev/smartguardian_app/build/ios/Debug-iphonesimulator/FirebaseMessaging/FirebaseMessaging.framework/Headers/FirebaseMessaging.h:18:9: warning: double-quoted include "FIRMessagingExtensionHelper.h" in framework header, expected angle-bracketed instead [-Wquoted-include-in-framework-header]
    #import "FIRMessagingExtensionHelper.h"
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            <FirebaseMessaging/FIRMessagingExtensionHelper.h>
    2 warnings generated.
: Error: Member not found: 'packageRoot'.
          io.Platform.packageRoot; // ignore: deprecated_member_use
                      ^^^^^^^^^^^
    Failed to package /Users/I540401/dev/smartguardian_app.
    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Planning
    note: Build preparation complete
    note: Building targets in dependency order
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/firebase_analytics/firebase_analytics.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/GoogleUtilities/GoogleUtilities.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/firebase_core/firebase_core.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/FirebaseCoreDiagnostics/FirebaseCoreDiagnostics.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/firebase_messaging/firebase_messaging.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/FirebaseInstallations/FirebaseInstallations.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/Pods_Runner.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/GoogleDataTransport/GoogleDataTransport.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/FirebaseCore/FirebaseCore.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/FirebaseMessaging/FirebaseMessaging.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/Runner.app'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/connectivity/connectivity.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/ImageNotification.appex'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/image_gallery_saver/image_gallery_saver.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/Reachability/Reachability.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/device_info/device_info.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/flutter_app_badger/flutter_app_badger.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/path_provider/path_provider.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/nanopb/nanopb.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/video_player/video_player.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/package_info/package_info.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/shared_preferences/shared_preferences.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/wakelock/wakelock.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/PromisesObjC/FBLPromises.framework'
    note: Removed stale file '/Users/I540401/Library/Developer/Xcode/DerivedData/Runner-hgbortgnmoyqfpesinxugifwbemc/Build/Products/Debug-iphonesimulator/permission_handler/permission_handler.framework'
    /Users/I540401/dev/smartguardian_app/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.4.99. (in target 'Reachability' from project 'Pods')
    Result bundle written to path:
    	/var/folders/82/y_h0djv17yvdwh0w89577w080000gn/T/flutter_tools.U4bujC/flutter_ios_build_temp_dirzJ47kt/temporary_xcresult_bundle
Could not build the application for the simulator.
Error launching application on iPhone 12 Pro.
Exited (sigterm)

```

**Comment by utytlanyjoe** [comment]
Found Solution.
1. Check for updates of XCode in App Store.
2. Launch XCode -> It will ask you to install additional software. Solved Problem for me.

**Comment by js1972** [comment]
@ste-el-bow you are one of the lucky ones. I'm already on xcode 13.3 so have nothing new to update to... Launching Xcode does not try to install any additional software.

**Comment by jinfagang** [comment]
@js1972 Same issue here. I have to mention that it just happened on some complicated project, a sample demo code doesn't have this issue. Also, seems only happened on running on Simulator. cc @darshankawar

**Comment by Lawati97** [comment]
Am getting the same issue even though am using an older version of flutter (2.8.0)

**Comment by js1972** [comment]
@jinfagang I get the same errors regardless of whether I'm building for the sim or my real phone plugged in.

**Comment by RCK-DEV** [comment]
In my case the error went away without upgrade macOS or changing anything else.. weird.

**Comment by Lawati97** [comment]
> 

https://stackoverflow.com/questions/70363918/error-member-not-found-packageroot-how-to-solve-ignore-deprecated-member-u

**Comment by Lawati97** [comment]
am getting this error

error: backend command failed due to signal 6 (use -v to see invocation)
    LLVM ERROR: out of memory
    Allocation failed

**Comment by darshankawar** [comment]
@Lawati97 
That looks unreleated to the issue being discussed in this thread. Please open new issue with all requested details in new issue template.

**Comment by darshankawar** [comment]
For the original issue reported by the OP @maxelerate , it was this error:

`failed extracting 'https://dl.google.com/firebase/ios/swiftpm/8.12.0/GoogleAppMeasurement.zip' which is required by binary target 'GoogleAppMeasurement':`

Whose solution could be https://stackoverflow.com/questions/70620150/flutter-error-googleappmeasurement-when-build-app-in-xcode

So everyone in this thread, if you are getting above error, please try the SO link and see if it helps.

**Comment by zanderso** [comment]
@stuartmorgan I think the original issue here sounds like it is covered by https://github.com/flutter/flutter/issues/98298#issuecomment-1072347143, but thought you should be aware in case there's something to integrate from that SO answer.

**Comment by jpcid** [comment]
In my case (I already have installed command line developer tools) I fix it by running

```
sudo xcode-select -r
```

**Comment by pkwiecien91** [comment]
Update version to **_2.10.3_** in project.
Next:
```
flutter clean
flutter pub get
```

**Comment by RTGRDH** [comment]
> Update version to **_2.10.3_** in project. Next:
> 
> ```
> flutter clean
> flutter pub get
> ```

Didn't work for me

**Comment by ryzizub** [comment]
I have solved the issue by downgrading XCode to 13.2 from 13.3

**Comment by workerbee22** [comment]
Also an issue for me:
- macOS Monterey v12.3
- Xcode 13.3
- Flutter v2.10.3

Have been trying a million things for 3 days ... no luck.

**Comment by js1972** [comment]
@ryzizub do you mind if I ask how you install an older version of xcode?

**Comment by ryzizub** [comment]
@js1972 sure thing. 

- I have downloaded previous version on [apple website](https://developer.apple.com/xcode/resources/)
- xib extracted to application folder 
- set right version using command `sudo xcode-select -switch /Applications/Xcode-13.2.app` (Xcode-13.2, is my naming in application folder, it up to you how to name it. You can also completely uninstall 13.3 a just downgrade)

After full setup you can make sure that you are using right version using command `/usr/bin/xcodebuild -version`

**Comment by fatadel** [comment]
@ryzizub Could you please also specify your Mac chip and the version of Flutter?

**Comment by ryzizub** [comment]
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.10.3, on macOS 12.2.1 21D62 darwin-x64, locale cs-CZ) 
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 13.2)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.65.2)
[✓] Connected device (1 available)
[✓] HTTP Host Availability

intel i7 6cores
```

**Comment by fatadel** [comment]
@ryzizub Thanx a lot, mate!

**Comment by fatadel** [comment]
@ryzizub That's so strange. I had almost the same set-up as you, except flutter version and M1 chip. So, I've upgraded flutter from 2.8.1 to 2.10.3 and the issue disappeared! 🤯 I still got XCode version 13.3.

**Comment by lipingx** [comment]
Everything was working fine. Then today after updating xcode to 13.3, I am getting this error.  
My macOS version: Moneterey version 12.2.1
Flutter (Channel stable, 2.8.1)
 
Launching lib/main.dart on iPhone SE (3rd generation) in debug mode...
Running Xcode build...                                                  
 └─Compiling, linking and signing...                         4.8s
Xcode build done.                                           53.2s
Failed to build iOS app
Error output from Xcode build:
↳
    2022-03-20 11:22:03.929 xcodebuild[23503:108094] Requested but did not find extension point with identifier
    Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in
    com.apple.dt.IDEWatchSupportCore
    2022-03-20 11:22:03.930 xcodebuild[23503:108094] Requested but did not find extension point with identifier
    Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension
    Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
    ** BUILD FAILED **


flutter doctor -v
[✓] Flutter (Channel stable, 2.8.1, on macOS 12.2.1 21D62 darwin-x64, locale en-US)
    • Flutter version 2.8.1 at /Users/lipingxiong/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 77d935af4d (3 months ago), 2021-12-16 08:37:33 -0800
    • Engine revision 890a5fca2e
    • Dart version 2.15.1

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/lipingxiong/Library/Android/sdk
    • Platform android-31, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] IntelliJ IDEA Community Edition (version 2019.3.1)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.35.0)
    • VS Code at /Users/lipingxiong/Documents/app_dev/Visual Studio Code.app/Contents
    • Flutter extension version 3.1.0

[✓] Connected device (2 available)
    • iPhone SE (3rd generation) (mobile) • A4595337-43AC-40B9-9B5B-FC4425D31EA0 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-15-4
      (simulator)
    • Chrome (web)                        • chrome                               • web-javascript • Google Chrome 99.0.4844.74

**Comment by ryzizub** [comment]
@lipingx Can you try downgrade to 13.2 if issue still remains, if yes, than we can be sure, that its something going on with flutter and xcode 13.3

**Comment by AshfaqAshi** [comment]
Here's how I solved the issue:
macOS: Monterey
XCode: 13.3
Flutter: 2.10
Dart : 2.16.1

Once you've the above config set, run 'flutter packages upgrade'.
Then create a backup of your 'ios' folder and delete it from your project repo.
now run 'flutter clean' followed by 'flutter create .' This will create an 'ios' folder in the repo.
Copy the Info.plist, Podfile and any other config files like GoogleServices.plist from the backup 'ios' folder to the newly created 'ios' folder.

Run the project now and it should work!

**Comment by lipingx** [comment]
@ryzizub instead of downgrade, I upgraded the Flutter. And the problem is fixed for me. 

It worked for me with the following setup:
macOS: Monterey 12.2.1
XCode: 13.3
Flutter: 2.10.3
Dart : 2.16.1

[✓] Flutter (Channel stable, 2.10.3, on macOS 12.2.1 21D62 darwin-x64, locale en-US)
    • Flutter version 2.10.3 at /Users/lipingxiong/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 7e9793dee1 (3 weeks ago), 2022-03-02 11:23:12 -0600
    • Engine revision bd539267b4
    • Dart version 2.16.1
    • DevTools version 2.9.2

**Comment by workerbee22** [comment]
Confirmed @lipingx and @AshfaqAshi I have matched your software levels.

But note, also on every machine I also had to clear our all my Cocoapods setup via this process: https://stackoverflow.com/questions/59362862/flutter-ios-build-failed-an-error-of-pod-files-podfile-is-out-of-date

After that local and CI/CD builds are fine.

**Comment by js1972** [comment]
Hi @AshfaqAshi I followed your solution and it has worked to at least build successfully for development. Thankyou very much!

One additional step I had to do was to uncomment the line at the top of the pods file:
```
# Uncomment this line to define a global platform for your project
platform :ios, '15.0'
```
I just set it to 15 - I think it defaulted to 9.

The saga continues for me though as it seems firebase have changed the way you need to initialise your app... so more reading and testing.  ;-)

**Comment by jmagman** [comment]
Looks like Xcode 13.3 prints `Requested but did not find extension point with identifier
                    Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in
                    com.apple.dt.IDEWatchSupportCore` on every build.  It is NOT an error, it looks like a harmless warning.  If you successfully build with `-v` you will also see it.

If you are seeing an error it is totally unrelated to that message and you need fix the actual error.  We've improved the Xcode error output as of https://github.com/flutter/flutter/pull/94747 in 2.10 but it looks like the original reporter is on 2.10.2.

**Can anyone seeing this on > Flutter 2.10 please attach the full output of the command you are running?**  All the examples I see are just truncated to the confusing Xcode warning text.  We'd like to improve this output further. 

We could suppress this message from Xcode to avoid confusion, but I suspect they will have fixed this by the time 13.3 GMs.

So to recap: that message is harmless and prints on every build regardless of whether it passes or fails.  This issue has become a conflation of many different build problems.  We can track that the confusing message is shown here, but you should file a new issue with the `--verbose` flag so we can actually try to diagnose your particular build issue.

**Comment by jmagman** [comment]
@cyanglaz now that #94747 is done can we get rid of the logic that prints the last x stderr lines from Xcode if the `xcResult` was successfully generated?

https://github.com/flutter/flutter/blob/42eb9032009b12a7671451b99a18828f3815b741/packages/flutter_tools/lib/src/ios/mac.dart#L388-L391

**Comment by cyanglaz** [comment]
> @cyanglaz now that #94747 is done can we get rid of the logic that prints the last x stderr lines from Xcode if the `xcResult` was successfully generated?

I think so. https://github.com/flutter/flutter/issues/100721 created

**Comment by xiaomaiTD** [comment]
<img width="1065" alt="434FCB10-7812-45A2-858C-24AF92BA5D92" src="https://user-images.githubusercontent.com/18552341/160090797-1ab1f323-86b2-480a-a6f0-8b59fd5380e9.png">
![38CB390F-C746-4858-804E-BFBBA15C654E](https://user-images.githubusercontent.com/18552341/160090807-6dbf761f-2824-4f0a-8553-d532e46dac7d.png)

**Comment by araafat3** [comment]
i solved issue by upgrade xcode to 13.3.1

**Comment by jmagman** [comment]
Xcode has stray logging every few releases, it's wack-a-mole suppressing them.

**Comment by namanathousy** [comment]
Same issue here @maxelerate @jmagman @darshankawar  
Using Mac OS Monterey 12.4 | Xcode Version- 13.4
Issue was the same since Xcode version 13.2, tried downloading dependencies as well. Nothing seems to work. Any help please?

Error output from Xcode build:
↳
    2022-05-25 18:30:32.893 xcodebuild[5688:64505] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension
    Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
    2022-05-25 18:30:32.894 xcodebuild[5688:64505] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension
    Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
    ** ARCHIVE FAILED **
    
    -> Command PhaseScriptExecution failed with a nonzero exit code

**Comment by gabrc52** [comment]
@jmagman On Flutter 3.0.1, I am getting this error only when I'm connected via SSH to the Mac for some reason (and every time when I do so), but I don't get it if I do `flutter run` from Terminal.app in the Mac. I suspect it might have to do with the difference in environment variables.

```
[rgabriel@vector ~]$ ssh 192.168.122.221
Last login: Wed Jun  8 04:03:06 2022 from 192.168.122.1
rgabriel@macos ~ % cd Projects
rgabriel@macos Projects % flutter create myapp --org com.notexample
Signing iOS app for device deployment using developer identity: "Apple Development: [redacted name] ([redacted team id])"
Creating project myapp...
Running "flutter pub get" in myapp...                              949ms
Wrote 127 files.

All done!
In order to run your application, type:

  $ cd myapp
  $ flutter run

Your application code is in myapp/lib/main.dart.

rgabriel@macos Projects % cd myapp
rgabriel@macos myapp % flutter run
Multiple devices found:
aifon (mobile)             • [redacted uuid] • ios        • iOS 14.8 18H17
iPhone 11 Pro Max (mobile) • 6FC1DB0A-8250-4FB4-99A2-3799989F7423     • ios        •
com.apple.CoreSimulator.SimRuntime.iOS-15-5 (simulator)
macOS (desktop)            • macos                                    • darwin-x64 • macOS 12.4 21F79 darwin-x64
[1]: aifon ([redacted uuid])
[2]: iPhone 11 Pro Max (6FC1DB0A-8250-4FB4-99A2-3799989F7423)
[3]: macOS (macos)
Please choose one (To quit, press "q/Q"): 1
Launching lib/main.dart on aifon in debug mode...
Automatically signing iOS for device deployment using specified development team in Xcode project: [redacted team id]
Running Xcode build...                                                  
Xcode build done.                                           16.5s
Failed to build iOS app
Error output from Xcode build:
↳
    2022-06-08 04:12:20.030 xcodebuild[56744:423826] Requested but did not find extension point with identifier
    Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of
    plug-in com.apple.dt.IDEWatchSupportCore
    2022-06-08 04:12:20.030 xcodebuild[56744:423826] Requested but did not find extension point with identifier
    Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension
    Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
    ** BUILD FAILED **


Xcode's output:
↳
    Writing result bundle at path:
    	/var/folders/zm/nrbt6lb11h50hp92r9bhg7mh0000gn/T/flutter_tools.z6BlUh/flutter_ios_build_temp_diruKzFzC/temporary_xc
    	result_bundle


    Failed to package /Users/rgabriel/Projects/myapp.
    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Planning
    note: Build preparation complete
    note: Building targets in dependency order

    Result bundle written to path:
    	/var/folders/zm/nrbt6lb11h50hp92r9bhg7mh0000gn/T/flutter_tools.z6BlUh/flutter_ios_build_temp_diruKzFzC/temporary_xc
    	result_bundle


Could not build the precompiled application for the device.

Error launching application on aifon.
```

Let me know if you need any additional information.

**Comment by mariapapag** [comment]
I'm also stuck on 


2022-06-08 04:12:20.030 xcodebuild[56744:423826] Requested but did not find extension point with identifier
    Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of
    plug-in com.apple.dt.IDEWatchSupportCore
    2022-06-08 04:12:20.030 xcodebuild[56744:423826] Requested but did not find extension point with identifier
    Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension
    Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
    ** BUILD FAILED **

I have command line tools installed 
I checked my xcode preferences > locations > command line tools is what it should be
I added the path of my command line tools in the zshrc file and that didn't work 

Please help :)
